src/views/v2/list/nav/next.php

```diff
@@ -22,7 +22,7 @@
 	<a
 		href="<?php echo esc_url( $link ); ?>"
 		rel="next"
-		class="tribe-common-c-nav__next"
+		class="tribe-common-c-nav__next tribe-events-navigation-link"
```

Comment:
any reason why we're not following BEM format with this?

Reply:
Not I was looking to figure out with you guys what would be the best formatting for this. I wasnt sure.